refgram/syntax/index.html

```diff
@@ -1659,7 +1659,7 @@ <h4 id="pronominal_classes"><a href="#pronominal_classes">Pronominal classes</a>
 		<div class="vocabularyg">/hoq</div>
 	</td>
 	<td>
-		<div class="vocabularyt">hóq</div>
+		<div class="vocabularyt">󱛆󱛊󱛃󱛂</div>
```

Comment:
And then this should remain romanized:
```suggestion
		<div class="vocabularyt">hóq</div>
```